matchms/filtering/add_retention.py

```diff
@@ -0,0 +1,83 @@
+from typing import Any
+from typing import List
+from matchms.utils import get_first_common_element
+from ..typing import SpectrumType
+
+
+_retention_time_keys = ["retention_time", "retentiontime", "rt", "scan_start_time"]

```

Comment:
One additional variation I came across was "RT_Query" (output of GNPS in MGF files)

Reply:
And "rtinseconds" in a mgf file

Reply:
Thank you very much for those valuable inputs!

Reply:
That's indeed good to know, thanks!

Reply:
Please see the additions in https://github.com/matchms/matchms/pull/265/commits/e22ed8f5278a6efe95e1146a3d334d1f7d3dc840